cv32e40s/tb/asserts/README.md

```diff
@@ -0,0 +1 @@
+Directory for assertion files and related models and support logic.
```

Comment:
I like this change. As far as I understand the goal here is to add our  assert files in this directory instead of the uvmt directory, right?
In that case, comment about the directory name:
As we know asserts and coverage is often written in the same file (as it often seems fit), so I would suggest to name the new directory assert_and_cov or something that at state that we also include coverage in these files.

But I beleive I might be mistaken about my interpretation ^. Because I dont understand why you have included the assertion files in the uvmt folder, and the model files in the assert folder... (I would at least think that the assert file should go into the assert folder)

Reply:
I don't think the number of files merits further division yet, although that might change in the future. Also the LRM argument makes sense, so i think the name "asserts" is the best option for the current structure.

Reply:
Okei, I see. Sounds good. You mean the name "assertion" and not "assert", right @silabs-mateilga? Since assertion is the common-term?

Reply:
correct

Reply:
Fixed.